### DISCUSSION TITLE
how can ı delete other connected school mail

### DISCUSSION THREAD

**Discussion by burakaltinbicak** [community/community#135010] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I forgot my old account and deleted it. But my school address was registered. I created new account but I can't use school address because already used.

**Comment by edwinm** [comment]
The only way to get your repos back is by contacting GitHub:

https://support.github.com/contact/account

**Comment by Chaitanya029** [comment]
Hi @burakaltinbicak 

Sorry to hear that you're having trouble with your GitHub account and school email address.

I understand that you forgot your old account and deleted it, but you're now unable to use your school email address with your new account because it's already associated with the old account.

Here are a few potential solutions you could try:

Try to recover your old account: If you remember any details about your old account, such as the username or password, you might be able to recover it. GitHub has a process for recovering deleted accounts, which you can find in their documentation.
Contact GitHub support: Reach out to GitHub support and explain your situation. They may be able to help you dissociate the school email address from your old account, allowing you to use it with your new account. Be prepared to provide proof of identity and verification that you're the same person who owned the old account.
Use a different email address: If you're unable to recover your old account or get the school email address dissociated, you could try using a different email address with your new GitHub account. This might not be ideal, but it would allow you to continue using GitHub with a different email address.
It's also worth noting that GitHub has policies in place to prevent email address reuse, so it's possible that you might not be able to use the same school email address with your new account.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @burakaltinbicak ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
